src/components/Register/Register.js

```diff
@@ -11,46 +21,71 @@ function RegisterPopup({ isOpen, onClose, onRedirect }) {
       isOpen={isOpen}
       onClose={onClose}
       onRedirect={onRedirect}
+      isValid={isValid}
+      email={values.email}
+      password={values.password}
+      username={values.username}
+      onRegisterSubmit={onRegisterSubmit}
+      resetForm={resetForm}
       buttonText="sign up"
+      isAuthError={isAuthError}
     >
       <div className="popup__field-container">
         <p className="popup__field-description">Email</p>
         <input
+          onChange={handleChange}
           name="email"
+          value={values.email || ""}
           autoComplete="off"
           className="popup__field popup__field-signup"
           type="email"
           placeholder="Enter email"
           required
         />
-        <div className="popup__error-container">
-          <span className="popup__field-error email-field-error"></span>
-        </div>
+        <span
+          className={`popup__field-error email-field-error" ${
+            isAuthError ? isAuthError : ""
+          }`}
+        >
+          {errors.email}
+        </span>
         <p className="popup__field-description">Password</p>
         <input
+          onChange={handleChange}
           name="password"
+          value={values.password || ""}
           autoComplete="off"
           className="popup__field popup__field-signup"
           type="password"
           placeholder="Enter password"
           required
           minLength="8"
         />
-        <div className="popup__error-container">
-          <span className="popup__field-error password-field-error"></span>
-        </div>
+        <span
+          className={`popup__field-error password-field-error" ${
+            isAuthError ? isAuthError : ""
+          }`}
+        >
+          {errors.password}
+        </span>
         <p className="popup__field-description">Username</p>
         <input
+          onChange={handleChange}
           name="username"
+          value={values.username || ""}
```

Comment:
Good job that you don't forget about `value` in the inputs if you have `onChange`